src/Components/FormPage/FormPage.css

```diff
@@ -100,8 +101,11 @@
 .form-input-container {
   width: 100%;
   display: flex;
+  height: auto;
+  overflow: scroll;
```

Comment:
I noticed that the bars where you would use to scroll are always appearing even if you don't need to scroll. I found if you instead use 'overflow-y: auto' it works the same without those blank bars appearing.

Reply:
Just fixed it!